src/raven.js

```diff
@@ -759,7 +760,14 @@ function makeRequest(data) {
     var img = newImage(),
         src = globalServer + authQueryString + '&sentry_data=' + encodeURIComponent(JSON.stringify(data));
 
-    img.crossOrigin = 'anonymous';
+    if ('crossOrigin' in globalOptions) {
```

Comment:
What's the point in checking this? You're setting it to a default in `globalOptions` already. So it should always exist, correct?